e2e_testing/xfail_sets.py

```diff
@@ -12,7 +12,10 @@
 
 from torch_mlir_e2e_test.test_suite import COMMON_TORCH_MLIR_LOWERING_XFAILS
 
-LINALG_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS
+LINALG_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS.union({
+    # TODO ERROR: dtype (torch.int8) is not equal to golden dtype (torch.uint8)
```

Comment:
XFAIL's are for unimplemented features. But this is a miscompile (the compiler succeeds, but the code runs and produces an incorrect result). It is absolutely never okay to knowingly commit a miscompile into the tree, since they are extremely difficult to debug if they ever reach production.

You will need to either implement this feature or guard it so that compilation fails with a useful diagnostic.

Reply:
Sure, how about I add an explicit error "unimplemented feature" into this PR, and attempt to provide a proper solution in the next PR?

Reply:
That sounds good to me.